salt/modules/zabbix.py

```diff
@@ -1452,3 +1452,50 @@ def hostinterface_update(interfaceid, **connection_args):
             raise KeyError
     except KeyError:
         return ret
+
+
+def template_get(name=None, host=None, templateids=None, **connection_args):
+    '''
+    Retrieve templates according to the given parameters.
+
+    Args:
+        host: technical name of the template
+        name: visible name of the template
+        hostids: ids of the templates
+
+        optional connection_args:
+                _connection_user: zabbix user (can also be set in opts or pillar, see module's docstring)
+                _connection_password: zabbix password (can also be set in opts or pillar, see module's docstring)
+                _connection_url: url of zabbix frontend (can also be set in opts or pillar, see module's docstring)
+
+                all optional template.get parameters: keyword argument names differ depending on your zabbix version, see:
+
+                https://www.zabbix.com/documentation/2.4/manual/api/reference/template/get
+
+    Returns:
+        Array with convenient template details, False if no template found or on failure.
+
+    CLI Example:
+    .. code-block:: bash
+
+        salt '*' zabbix.template_get name='Template OS Linux'
+        salt '*' zabbix.template_get templateids="['10050', '10001']"
+    '''
+    conn_args = _login(**connection_args)
+    try:
+        if conn_args:
+            method = 'template.get'
+            params = {"output": "extend", "filter": {}}
+            if 'name':
```

Comment:
`if name:`

Please remove the quotes. Otherwise it always returns True because it just checks the existence of the defined string.